File: backend/art-galery/src/main/java/com/unmsm/movil/tecnisis/art_galery/infrastructure/adapters/input/rest/model/response/EconomicEvaluationResponse.java
package com.unmsm.movil.tecnisis.art_galery.infrastructure.adapters.input.rest.model.response;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDate;

@Builder @Getter @Setter
@AllArgsConstructor @NoArgsConstructor
public class EconomicEvaluationResponse {
    private Long id;
    private LocalDate evaluationDate;
    private BigDecimal salesPrice;
    private BigDecimal galleryPercentage;
    private String status;
    private SpecialistResponse specialist;
    private RequestResponse request;
    private DocumentResponse document;
}

File: tecnisis-app/app/src/main/java/com/example/tecnisis/data/evaluations/EconomicEvaluationResponse.kt
package com.example.tecnisis.data.evaluations

import com.example.tecnisis.data.document.DocumentResponse
import com.example.tecnisis.data.specialist.SpecialistResponse
import com.google.gson.annotations.SerializedName

data class EconomicEvaluationResponse(
    val id: Long,
    @SerializedName("evaluation_date")
    val evaluationDate: String,
    @SerializedName("sales_price")
    val salesPrice: Double,
    @SerializedName("gallery_percentage")
    val galleryPercentage: Double,
    val status: String,
    val specialist: SpecialistResponse,
    val document: DocumentResponse
)
